codespell_lib/data/dictionary.txt

```diff
@@ -46733,6 +46801,8 @@ quantititive->quantitative
 quantitity->quantity
 quantitiy->quantity
 quarantaine->quarantine
+quarantee->guarantee
```

Comment:
very close to 'quarantine', could be a mistake for a non-english speaker perhaps ?

Reply:
I haven't found such a typo out there:
https://grep.app/search?q=quarantee

Reply:
just out of curiosity i through it into google:

![Screenshot_20240909_071018](https://github.com/user-attachments/assets/eab9ff68-b20d-4af7-be00-6aa8f6f9760f)

Reply:
Such dictionary suggestions are based on a distance metric between words, but that distance metric doesn't necessarily reflect the probability of typos in practice. Anyway, I have added `quarantine`.